assets/js/base/components/form-token-field/style.scss

```diff
@@ -0,0 +1,58 @@
+.wc-blocks-components-form-token-field-wrapper {
+	border-color: $gray-600;
+
+	.components-visually-hidden,
+	.components-form-token-field__label {
+		@include visually-hidden();
+	}
+
+	.components-form-token-field,
+	.components-form-token-field__input-container {
+		border-color: inherit;
+		box-sizing: border-box;
+		margin-bottom: 0;
+	}
+
+	.components-form-token-field__input-container {
+		border-radius: 0;
+		box-shadow: none;
+		position: relative;
+
+		input[type="text"].components-form-token-field__input {
+			min-height: 30px;
+		}
+	}
+
+	.components-form-token-field__remove-token {
+		border: none;
+	}
+
+	.components-form-token-field__suggestions-list {
+		background-color: $white;
+		border: none;
+		left: -1px;
+		margin: 1px 0 0;
+		position: absolute;
+		right: -1px;
+		top: 100%;
+
+		> li {
+			border: 1px solid $gray-600;
+			border-width: 0 1px;
+		}
+
+		> li:last-child {
+			border-width: 0 1px 1px;
+		}
```

Comment:
Small suggestion: I would move the border from the list items into the wrapper, so it's always visible even if the list overflows. I also needed to add a `z-index` value to avoid other elements overlapping it:

Before | After
--- | ---
![imatge](https://user-images.githubusercontent.com/3616980/179493788-acf0084c-20ea-48e0-9a07-a35a4e3e1192.png) | ![imatge](https://user-images.githubusercontent.com/3616980/179493699-978275f5-0bd4-4387-82ae-1ac1343afbf7.png)

```suggestion
		border: 1px solid $gray-600;
		border-width: 0 1px 1px;
		z-index: 1;

		&:empty {
			display: none;
		}
```

Adding `display: none` when the element is empty makes it so the border is not visible if there are no suggestions.

Reply:
> Adding `display: none` when the element is empty makes it so the border is not visible if there are no suggestions.

TIL! Thanks for the lesson on the `:empty` selector!

Reply:
FYI, I use `50` for `z-index` because the nobs of the price filter use `21`. We can use `25` but I think `50` is more future proof.

Reply:
Sounds good!